DOCS-1717: Fix hyperlinks to payment method icons (#940)
MultiSafepay/docs: content/faq/general/where-find-logo-payment-methods.md

read_more: "."
---

> You can find the icons of the payment methods in your MultiSafepay Control on [this page](https://merchant.multisafepay.com/payment-logo-designer)

Author: ```suggestion
You can find the icons of the payment methods in your MultiSafepay Control [here](https://merchant.multisafepay.com/payment-logo-designer)
```
